Gems/Atom/Feature/Common/Code/Source/FrameCaptureSystemComponent.h

```diff
@@ -47,6 +49,22 @@ namespace AZ
             uint32_t CapturePassAttachmentWithCallback(const AZStd::vector<AZStd::string>& passHierarchy, const AZStd::string& slotName
                 , RPI::AttachmentReadback::CallbackFunction callback, RPI::PassAttachmentReadbackOption option) override;
 
+            void SetScreenshotFolder(const AZStd::string& screenshotFolder) override;
+            void SetTestEnvPath(const AZStd::string& envPath) override;
+            void SetOfficialBaselineImageFolder(const AZStd::string& baselineFolder) override;
+            void SetLocalBaselineImageFolder(const AZStd::string& baselineFolder) override;
+
+            AZStd::string BuildScreenshotFilePath(const AZStd::string& imageName) override;
+            AZStd::string BuildOfficialBaselineFilePath(const AZStd::string& imageName) override;
+            AZStd::string BuildLocalBaselineFilePath(const AZStd::string& imageName) override;
```

Comment:
I think you need an `// implementing bus` block comment above this.

Reply:
Fixed